datacapturing/src/main/java/de/cyface/datacapturing/backend/CapturingProcess.java

```diff
@@ -265,9 +262,7 @@ public synchronized void onSensorChanged(final @NonNull SensorEvent event) {
                 rotations.clear();
                 directions.clear();
                 lastNoGeoLocationFixUpdateTime = thisSensorEventTime;
-            } catch (SecurityException e) {
-                throw new IllegalStateException(e);
-            } catch (DataCapturingException e) {
+            } catch (SecurityException | DataCapturingException e) {
```

Comment:
Probably should move changes for updates due to newer Android version to another PR.